### PR TITLE
DOCSP-15886 doc limitations with shared clusters

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
## DESCRIPTION
Adds documentation for mongodump and mongorestore limitations with free and shared Atlas clusters

## STAGING

- https://preview-mongodbkanchanamongodb.gatsbyjs.io/database-tools/DOCS-15886/mongodump/#using-mongodump-on-atlas-cluster
- https://preview-mongodbkanchanamongodb.gatsbyjs.io/database-tools/DOCS-15886/mongorestore/#using-mongorestore-on-atlas-cluster

## JIRA
https://jira.mongodb.org/browse/DOCS-15886

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65c5568e6b2f17c95a52c551

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/mongodump.txt [hunk 1]
```diff
@@ -372,6 +372,21 @@ FIPS
 connections to a :binary:`~bin.mongod`/:binary:`~bin.mongos` that is
 :manual:`configured to use FIPS mode </tutorial/configure-fips>`.
 
+Using ``mongodump`` on Atlas Cluster
```
Comment (ajhuh-mdb) [inline source/mongodump.txt:375]: [suggestion] This applies to the mongorestore page, too, but for grammar, I think we either want to drop "cluster," make it plural into "Atlas Clusters," or add "an" before "Atlas Cluster".

```suggestion
Using ``mongodump`` on Atlas Clusters 
```

File: source/mongorestore.txt [hunk 2]
```diff
@@ -215,6 +215,22 @@ Starting in MongoDB 5.0, you can use ``mongorestore`` to restore
 :ref:`timeseries collections <manual-timeseries-landing>`.
 For details, see :ref:`mongorestore-example-time-series`.
 
+Using ``mongorestore`` on Atlas Cluster
```
Comment (ajhuh-mdb) [inline source/mongorestore.txt:218]: Same suggestion as above

**Review by ajhuh-mdb — APPROVED** [review]
LGTM % one suggestion

File: source/mongodump.txt [hunk 3]
```diff
@@ -372,6 +372,21 @@ FIPS
 connections to a :binary:`~bin.mongod`/:binary:`~bin.mongos` that is
 :manual:`configured to use FIPS mode </tutorial/configure-fips>`.
 
+Using ``mongodump`` on Atlas Clusters
```
Comment (rohshar) [inline source/mongodump.txt:375]: This title should probably say Atlas free and shared tier clusters

File: source/mongorestore.txt [hunk 4]
```diff
@@ -215,6 +215,22 @@ Starting in MongoDB 5.0, you can use ``mongorestore`` to restore
 :ref:`timeseries collections <manual-timeseries-landing>`.
 For details, see :ref:`mongorestore-example-time-series`.
 
+Using ``mongorestore`` on Atlas Clusters 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On free (``M0``) and shared (``M2`` and ``M5``) tier Atlas clusters, the
+following limitations apply:
+
+- You can't run ``mongorestore`` on the ``admin`` database. If you use
```
Comment (rohshar) [inline source/mongorestore.txt:224]: I think it should also mention that it is skipped by default (if db unspecified) on these clusters

File: source/mongorestore.txt [hunk 5]
```diff
@@ -215,6 +215,23 @@ Starting in MongoDB 5.0, you can use ``mongorestore`` to restore
...
+Using ``mongorestore`` on Atlas Free and Shared Tier Clusters 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On free (``M0``) and shared (``M2`` and ``M5``) tier Atlas clusters, the
+following limitations apply:
+
+- You can't run ``mongorestore`` on the ``admin`` database. If you use
+  the :option:`--db <mongorestore --db>` option to set the destination
+  database to ``admin``, the program returns an error. By default,
+  ``mongorestore`` skips this option.
```
Comment (rohshar) [inline source/mongorestore.txt:227]:
```suggestion
  ``mongorestore`` skips this database.
```

Comment (rohshar) [reply source/mongorestore.txt:227]: I would also add this sentence above the previous one.

Also, I think that you should add a similar sentence to the mongodump section

**Review by rohshar — CHANGES_REQUESTED** [review]

**Review by rohshar — APPROVED** [review]

**Pull request merged**